### ISSUE TITLE
Unable to Collect Milk from Cows

### ISSUE THREAD

**Issue by MuteTiefling** [TeamPneumatic/pnc-repressurized#611]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

### Minecraft Version
1.16.1
### Forge Version
32.0.108
### Mod Version
pneumaticcraft-repressurized-1.16.1-2.2.1-32

### Describe your problem, including steps to reproduce it

Buckets appear to be voided when attempting to use the Right Click Entity ProgWidget to gather milk buckets

### Any other comments?

Test Program: https://gist.github.com/MuteTiefling/f35c9d229af8fe4cdec86c696c4a4a3d

Video: https://streamable.com/hclwrp

What I'm seeing here that really sticks out as strange... it still seems to show the empty buckets in the inventory of the drone... but when I wrench it I get nothing back, so the inventory appears to be truly empty. 

In the recording I had a slightly older version of PNC installed, as well as Mekanism and Botania, but I went back and updated PNC / removed the other mods and it's still doing it.

**Comment by desht** [comment]
Yep, confirmed.  The drone fake player's inventory wasn't being properly copied back to the drone.

**Comment by desht** [comment]
Should be fixed in build 33 (along with the death message translations) if you'd like to test it out...

Feel free to test corner cases too (e.g. drone holding multiple buckets without any inv upgrades).  That should be handled gracefully (drone drops excess milk buckets on the ground), but extra testing is always good :)

**Comment by MuteTiefling** [comment]
Yep, looking good. 

A drone with inventory upgrades successfully gets the milk in its inventory. A drone without them creates a fun little popcorn machine with buckets flying everywhere. 

I couldn't think of any other cases with buckets, but I figured I'd double check other right click functions. So I made a pretty fun sheep popcorn machine and also an animal breeder. All good on all fronts. 

I did notice a minor issue with the entity filters though. In both the F1 help and the patchouli manual, the specifiers are listed with spaces in them, but the text widget says that's invalid when entered. 

For instance, the following are listed with spaces, but to use them in the text widget, the spaces need to be removed. Again, very minor. 

@animal(age = adult)
@animal(age = baby)
@animal(breedable = yes)

Also, it could be nice to have those listed somehow directly in the text widget. Maybe a shortcut to pertinent filter pages in the Text Widget entry in patchouli would be appropriate. Go to info on the Text Widget opens the manual and you're presented with both the list of widgets it can be used with (as currently) and a link to the Variables and Filters entries as well, so you can go straight to them without having to leave the programming interface.

**Comment by desht** [comment]
> Also, it could be nice to have those listed somehow directly in the text widget. Maybe a shortcut to pertinent filter pages in the Text Widget entry in patchouli would be appropriate.

I'd probably rather go with the "hold F1 for help" you get elsewhere when entering an entity filter, although in this case I would also want to check that this text widget is for an entity filter and not something else.  *Should* be possible based on the widget to the left of the text widget, but I'll see how feasible it is...

**Comment by desht** [comment]
Fix is in 2.2.2.  Gonna keep this open to remind me about the help text bit.
